hub/apps/get-started/windows-developer-faq.yml

```
@@ -33,12 +34,12 @@ sections:
     questions:
       - question: What is the Windows App SDK?
         answer: |
-          The Windows App SDK is a Windows application development platform that allows you to create beautiful, modern, backwards-compatible (down to Win10 1809) desktop apps. WinUI 3 is the UI framework that ships with Windows App SDK.
+          The Windows App SDK is a Windows application development platform that allows you to create beautiful, modern, backwards-compatible (down to Windows 10 1809) desktop apps. WinUI 3 is the UI framework that ships with Windows App SDK.
       - question: What's the difference between the Windows App SDK and the Windows SDK?
         answer: |
           Both are software development kits (SDKs) that let you build Windows apps.  
 
-          The **Windows App SDK** is a new development platform that lets you build modern desktop apps that can be installed across Windows versions (down to Win10 1809). When you use Windows App SDK to build an app, you’ll have access to the latest Windows development platform features. The Windows App SDK includes WinUI 3.
+          The **Windows App SDK** is a new development platform that lets you build modern desktop apps that can be installed across Windows versions (down to Windows 10 1809). When you use Windows App SDK to build an app, you’ll have access to the latest Windows development platform features. The Windows App SDK includes WinUI 3.
```

Prefer timeless reference, tweaked wording.

```suggestion
          The **Windows App SDK** helps you build modern desktop apps that can be installed across Windows versions (down to Windows 10 1809). When you use Windows App SDK to build an app, you’ll have access to the latest Windows development platform features. The Windows App SDK includes WinUI 3.
```